From b1925f54cff9c2bc93efbe553234656f00b7223c Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedricx.bondier@intel.com>
Date: Mon, 7 Jan 2013 16:07:34 +0100
Subject: [PATCH 10/12] Multivendor support: adding libbt-vendor conf for x86
 targets

BZ: 76413

Including the specific Intel configuration file for all x86 targets
in libbt-vendor configuration

Change-Id: I788675811f96b45e28b00c18cf597ad3025fd195
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-on: http://android.intel.com:8080/85372
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Biselli, MirkoX <mirkox.biselli@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 libbt/conf/intel/common/Android.mk     |   15 +++++++++++++++
 libbt/conf/intel/common/bt_vendor.conf |    5 +++++
 2 files changed, 20 insertions(+), 0 deletions(-)
 create mode 100644 libbt/conf/intel/common/Android.mk
 create mode 100644 libbt/conf/intel/common/bt_vendor.conf

diff --git a/libbt/conf/intel/common/Android.mk b/libbt/conf/intel/common/Android.mk
new file mode 100644
index 0000000..fd8d125
--- /dev/null
+++ b/libbt/conf/intel/common/Android.mk
@@ -0,0 +1,15 @@
+LOCAL_PATH := $(call my-dir)
+
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := bt_vendor.conf
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := ETC
+LOCAL_MODULE_PATH := $(TARGET_OUT)/etc/bluetooth
+
+LOCAL_MODULE_TAGS := eng
+
+LOCAL_SRC_FILES := $(LOCAL_MODULE)
+
+include $(BUILD_PREBUILT)
+
diff --git a/libbt/conf/intel/common/bt_vendor.conf b/libbt/conf/intel/common/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/common/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
-- 
1.7.9

